pdns/recursordist/rec-zonetocache.cc

```diff
@@ -263,7 +263,7 @@ void ZoneData::ZoneToCache(const RecZoneToCache::Config& config, uint64_t config
 // Config must be a copy, so call by value!
 void RecZoneToCache::ZoneToCache(RecZoneToCache::Config config, uint64_t configGeneration)
 {
-  setThreadName("pdns-r/ztc/" + config.d_zone);
+  setThreadName("ztc:" + config.d_zone);
```

Comment:
This buys `7` chars for the `ztc` zone portion of the 15 char thread name, which is a pretty sizable budget reallocation.